eogrow/pipelines/byoc.py

```diff
@@ -27,25 +28,40 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+TIMESTAMP_FORMAT = "%Y-%m-%dT%H-%M-%S"  # TODO: import from .export_maps once MRs are synced
 
 
 class IngestByocTilesPipeline(Pipeline):
     """Ingests .tiff files to a BYOC collection. Based on `ExportMapsPipeline` output for timeless features."""
 
     class Schema(Pipeline.Schema):
         byoc_tile_folder_key: str
-        file_glob_pattern: str = Field("**/UTM_*/*.tiff", description="Pattern used to obtain the TIFF files to use")
+        file_glob_pattern: str = Field("**/*.tiff", description="Pattern used to obtain the TIFF files to use")
 
         new_collection_name: Optional[str] = Field(description="Used for defining a new BYOC collection.")
         existing_collection: Optional[DataCollection] = Field(description="Used when updating and reingesting.")
         _parse_byoc_collection = optional_field_validator("existing_collection", parse_data_collection, pre=True)
         _ensure_exclusion = ensure_exactly_one_defined("new_collection_name", "existing_collection")
 
-        sensing_time: datetime.datetime = Field(description="Sensing time (ISO format) added to BYOC tiles.")
+        is_temporal: bool = Field(
```

Comment:
How is the timestamp from the parent folder inferred? should the user follow some specific instruction on how to produce the storage structure to successfully infer the timestamps?